--- install_amp	2008-09-10 15:44:06.000000000 -0500
+++ install_amp	2008-09-10 15:44:32.000000000 -0500
@@ -896,18 +896,18 @@
 
 // **** Make sure selinux isn't enabled
 
-outn("Checking for selinux..");
-$tmpoutput = array();
-$tmpout = exec("getenforce 2>/dev/null", $tmpoutput, $sereturn);
-if (strtolower($tmpoutput[0]) === "enabled") {
-        // this method seems better because disabled and permissive are the same
-        // if a user installs and realizes selinux is running the other method 
-        // requires a reboot to get selinuxenabled to work after editing the  selinux config
-        // this will allow you to use setenforce 0 which turns selinux into permissive mode which
-        // doesnt enforce, it just warns.
-	fatal("selinux is ENABLED. This is not supported. Please disable selinux before using FreePBX");
-}
-out("OK");
+#outn("Checking for selinux..");
+#$tmpoutput = array();
+#$tmpout = exec("getenforce 2>/dev/null", $tmpoutput, $sereturn);
+#if (strtolower($tmpoutput[0]) === "enabled") {
+#        // this method seems better because disabled and permissive are the same
+#        // if a user installs and realizes selinux is running the other method 
+#        // requires a reboot to get selinuxenabled to work after editing the  selinux config
+#        // this will allow you to use setenforce 0 which turns selinux into permissive mode which
+#        // doesnt enforce, it just warns.
+#	fatal("selinux is ENABLED. This is not supported. Please disable selinux before using FreePBX");
+#}
+#out("OK");
 
 // **** Connect to database
 
